packages/extension-api/src/extension-api.d.ts

```diff
@@ -308,7 +308,7 @@ declare module '@podman-desktop/api' {
    * It can be turned off/on by the user.
    */
   export interface ProviderAutostart {
-    start(logger: Logger): Promise<void>;
+    start(logger?: Logger): Promise<void>;
```

Comment:
What is the rationale for this change ? Are there some new cases where the logger was not being passed ?

Reply:
removed. I guess it was some old thing i forgot to remove